### DISCUSSION TITLE
Packagist misses dependency information

### DISCUSSION THREAD

**Discussion by fnagel** [deployphp/deployer#3300] [General]
Looks like Packagist won't use the dependency information within the composer.json file:
![image](https://user-images.githubusercontent.com/116824/189877832-b6c8f834-3e40-40b0-b386-c5caf29ab619.png)

Version 7.0.0-rc4 ist the last one with those information:
![image](https://user-images.githubusercontent.com/116824/189877949-fc1c5c06-cc3e-433d-b212-1b9f28e1d704.png)

**Comment by antonmedv** [comment]
This is by design. All deps inside phar.

**Reply by fnagel** [reply]
Ahh ok, thanks for the info!
